### ISSUE TITLE
Accessibility: Color Uplift

### ISSUE THREAD

**Issue by erinplourde** [cerner/carbon-graphs#193]
Carbon Colors need to work to meet new accessibility guidelines. Colors must meet a contrast ratio of 3:1.  https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

The recommended update to the color palette is: 
<img width="852" alt="Screen Shot 2020-04-06 at 1 28 43 PM" src="https://user-images.githubusercontent.com/52933964/78592317-8b485180-780a-11ea-968e-0722ee1c9860.png">

[](https://wiki.cerner.com/pages/viewpage.action?spaceKey=UserExperience&title=Carbon%20Graphing%20Library&preview=/2010049856/2225175269/Carbon_Colors.pdf)

The palette removes yellow and adds pink in its place. 

This issue is linked to: #15

**Comment by erinplourde** [comment]
UX Documentation: https://wiki.cerner.com/pages/viewpage.action?spaceKey=UserExperience&title=Carbon%20Graphing%20Library&preview=/2010049856/2225175269/Carbon_Colors.pdf

**Comment by erinplourde** [comment]
This color should be added to the palette (this will give consumers 7 distinct colors) 
<img width="89" alt="Screen Shot 2020-05-27 at 4 00 15 PM" src="https://user-images.githubusercontent.com/52933964/83071707-2acbd880-a033-11ea-975f-5b769c93a145.png">

cc @laurenbock

**Comment by AshishMotanamGurunadham** [comment]
@erinplourde  May i know what should i name this color?

cc @Veera81  @laurenbock

**Comment by erinplourde** [comment]
@AshishMotanamGurunadham we have been calling it lavender 

cc @Veera81  @laurenbock

**Comment by Veera81** [comment]
Thanks, @erinplourde 

@AshishMotanamGurunadham  Can you add lavender to [carbon color Palette](https://github.com/cerner/carbon-graphs/pull/214) and update the documentation.

**Comment by qinfen86** [comment]
@AshishMotanamGurunadham @Veera81 hey guys can I know when this branch can be merged? It's blocking some of our work and we're waiting for lavender to be added. Thanks for the help!

**Comment by Veera81** [comment]
Hey Qin,

We are adding Lavender color to the PR shortly. 

Once UX team review the PR and We will ready to merge the PR.

cc: @erinplourde

**Comment by Veera81** [comment]
@erinplourde 

Can you please review this PR when you get a chance today: https://github.com/cerner/carbon-graphs/pull/214

We will merge the PR once the review is done.

**Comment by chrismichalewicz** [comment]
Added the Requirements Needed tag as there are requirements that reference the color palate and will need to be updated once the updated color palate has been finalized.

**Comment by chrismichalewicz** [comment]
Updated the [requirements ](https://jazz.cerner.com:9443/rm/web#action=com.ibm.rdm.web.pages.showArtifact&artifactURI=https%3A%2F%2Fjazz.cerner.com%3A9443%2Frm%2Fresources%2FMB_1510aa237f5d4a2897f4c86738988fe8&vvc.configuration=https%3A%2F%2Fjazz.cerner.com%3A9443%2Frm%2Fcm%2Fstream%2F_Xq8-EJw6Eem70egXDYwlnA&componentURI=https%3A%2F%2Fjazz.cerner.com%3A9443%2Frm%2Frm-projects%2F_IQjCYFZ2EeWZLe9RyS0isA%2Fcomponents%2F_QKI4AJw6Eem70egXDYwlnA)to include the new color palate and hex codes. Removing the Requirements needed label.

**Comment by Veera81** [comment]
+1 for Requirements
